MacrobenchmarkSample/macrobenchmark/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    defaultConfig {
+        minSdkVersion 29
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
this should be default runner, not needed

Reply:
We need this for otherwise our tests don't seem to get picked up at all.
Also our documentation specifically calls out that we add this.
(https://developer.android.com/training/testing/unit-testing/instrumented-unit-tests)

Reply:
Ah, I've been spoiled by benchmark plugin and or androidx plugin adding the right runner for you